packages/styleguide/src/components/Form/docs.md

```diff
@@ -59,6 +59,20 @@ state: {
   onDec={() => setState({value: state.value - 10})} />
 ```
 
+### Clear field
+
+```react
+state: {
+  value: 'Eine Rolle'
+}
+---
+<Field
+  label='Rolle'
+  value={state.value}
+  onChange={(_, value) => setState({value})}
+  onClear={() => setState({value:""})} />
```

Comment:
do we need a separate function? can we just call `onChange` with an empty string? `onChange` always gets a string anyhow. Also see `browserIconStyle` – needs to be adapted to ensure native icon (like native clear is disabled when we explicitly make a clear button)